docs/content/v0.20/packaging.md

```diff
@@ -9,48 +9,135 @@ Note that to _use_ KubeStellar, all of the repos in this graph are involved.
-into the kubestellar GitHub organization; the actual dependency right
-now is on [a private repo](#status-addon).
-
 ## KubeFlex
 
 See [the GitHub repo](https://github.com/kubestellar/kubeflex).
 
-## status-addon
+## OCM Status Addon
+
+The [OCM Status Addon](https://github.com/kubestellar/ocm-status-addon) repo is the source of a RedHat-style operator.
+
+### Outline of OCM status addon publishing
+
+```mermaid
+flowchart LR
+    subgraph "ocm-status-addon@GitHub"
+    osa_code[OSA source code]
+    osa_hc_src[OSA Helm chart source]
+    end
+    osa_ctr_image[OSA container image] --> osa_code
+    osa_hc_repo[published OSA Helm Chart] --> osa_hc_src
+    osa_hc_src -.-> osa_ctr_image
+    osa_hc_repo -.-> osa_ctr_image
+```
+
+The dashed dependencies are at run time, not build time.
+
+"OSA" is OCM Status Addon
+
+### OCM status addon container image
+
+There is a container image at [ghcr.io/kubestellar/ocm-status-addon](https://github.com/orgs/kubestellar/packages/container/package/ocm-status-addon).
+
+The container image is built and published using `make`.  The
+following `make` variables are relevant.
+
+- `KO_DOCKER_REPO`, defaults to `ghcr.io/kubestellar`
+- `CMD_NAME`, defaults to `ocm-status-addon`
+- `IMAGE_TAG`, defaults to `0.2.0-alpha.1`
+- `IMG`, defaults to `${KO_DOCKER_REPO}/${CMD_NAME}:${IMAGE_TAG}`
+
+`make ko-build-push` will build and publish the container image at
+`${IMG}`. This will be a multi-platform manifest, referencing images
+for "platforms" `linux/amd64` and `linux/arm64`. This will not work if
+you have `podman` aliased to `docker`.
+
+`make ko-local-build` will build the same multi-platform manifest but
+not push it, only leave it among your Docker images. This also will
+not work if you have `podman` aliased to `docker`.
+
+
+### OCM status addon Helm chart
+
+The operator is delivered by a Helm chart at [ghcr.io/kubestellar/ocm-status-addon-chart](https://github.com/orgs/kubestellar/packages/container/package/ocm-status-addon-chart).
 
-The [status-addon](https://github.ibm.com/dettori/status-addon) repo is the source of a RedHat-style operator.
+The chart is built and published using `make`. The following `make`
+variables are relevant.
 
-The operator is delivered by a Helm chart at `quay.io/pdettori/status-addon-chart`. Chart versioning? How is the chart built, pushed?
+- `KO_DOCKER_REPO`, defaults to `ghcr.io/kubestellar`
+- `CMD_NAME`, defaults to `ocm-status-addon`
+- `IMAGE_TAG`, defaults to `0.2.0-alpha.1`
+- `IMG`, defaults to `${KO_DOCKER_REPO}/${CMD_NAME}:${IMAGE_TAG}`
 
-There is also a container image involved.
+`make chart`: builds the local copy of the chart from local sources,
+using `kustomize`. When using the default values of the `make`
+variables, the built chart references the container image
+`ghcr.io/kubestellar/ocm-status-addon:0.2.0-alpha.1`. TODO: describe
+how the chart contents and destination depend on the `make` variables.
+
```

Comment:
The image reference is updated when `make chart` is run by kustomize using the IMG variable

Reply:
That will be easier to follow once https://github.com/kubestellar/ocm-status-addon/issues/9 is fixed.